code/game/turfs/plating.dm

```diff
@@ -48,6 +48,13 @@
 	icon_state = "plating"
 	name = "airless plating"
 
+/turf/open/floor/plating/fake_space
+	icon = 'icons/turf/space.dmi'
+	name = "\proper space"
+	light_power = 0.25
+	icon_state = "0"
+	plane = PLANE_SPACE
+
```

Comment:
We shouldnt give a bunch of tiles lighting like this,it can be a perf hog, cant you just use something like baselighting (area based lighting) instead

Reply:
Yeah sure I can completely remove the light power var. The reason I added it was because that's what space tiles have, and I wanted the fake space tiles to look more authentic. 
Would me removing the var pass your requested change?

Reply:
Removed light var per request on discord.